index.html

```diff
@@ -0,0 +1,171 @@
+        <a href="checkout.html" class="button">Checkout</a>
+      </div>
+    </header>
+    <nav class="global-nav">
+      <a class="header-link" href="index.html">Home /</a>
+      <a class="header-link" href="dogs.html">  &nbsp;&nbsp;Dogs  /</a>
+      <a class="header-link" href="blog.html">  &nbsp;&nbsp;Blog    /</a>
+    </nav>
+
+    <section class="main">
+      <div class="grid-item-welcome">
+        <img class="welcome-image" src="images/home-intro.jpg" alt="a guy hugging his dog"><br>
+        <article class="about-text">
+          <h2>Adopt A Dog</h2>
+          <p>Cras sit amet purus nec felis dapibus tempus. Phasellus maximus metus ut tellus gravida, a pharetra ex posuere. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Etiam quis elit sapien. Donec nec sodales nulla. Praesent placerat tellus id tortor feugiat eleifend. Integer lacinia rhoncus diam faucibus sodales. Morbi posuere massa ac tempus pellentesque. Maecenas posuere vehicula nunc, sed bibendum massa tincidunt quis. Maecenas nec dignissim odio. Cras mollis magna nec lobortis bibendum. Aliquam ut felis sit amet risus aliquam pellentesque.</p>
+        </article>
+      </div>
+
+      <div class="dog-container">
+        <article class="dog">
+          <img src="images/murphy-card.jpg" alt="a basset hound" onclick="alertDogInfo('Murphy', 'Basset Hound', 123.45)">
+          <h4>Murphy</h4>
+          <p>Cost to Adopt: $123.45</p>
+          <p>Cras sit amet purus nec felis dapibus tempus.</p>
+          <button class="dog-button" onclick="addToTotal(123.45)">Adopt</button>
+        </article>
+
+        <article class="dog">
+          <img src="images/poppy-card.jpg" alt="a small fluffy gray dog" onclick="alertDogInfo('Poppy', 'Shihtzu', 123.45)">
+          <h4>Poppy</h4>
+          <p>Cost to Adopt: $123.45</p>
+          <p>Cras sit amet purus nec felis dapibus tempus.</p>
+          <button class="dog-button" onclick="addToTotal(123.45)">Adopt</button>
+        </article>
+
+        <article class="dog">
+          <img src="images/jack-card.jpg" alt="a white and brown dog" onclick="alertDogInfo('Jack', 'Beagle', 123.45)">
+          <h4>Jack</h4>
+          <p>Cost to Adopt: $123.45</p>
+          <p>Cras sit amet purus nec felis dapibus tempus.</p>
+          <button class="dog-button" onclick="addToTotal(123.45)">Adopt</button>
+        </article>
+
+        <article class="dog">
+          <img src="images/duffy-card.jpg" alt="a white and brown pitbull" onclick="alertDogInfo('Duffy', 'Pitbull', 123.45)">
+          <h4>Duffy</h4>
+          <p>Cost to Adopt: $123.45</p>
+          <p>Cras sit amet purus nec felis dapibus tempus.</p>
+          <button class="dog-button" onclick="addToTotal(123.45)">Adopt</button>
+        </article>
+
+        <article class="dog">
+          <img src="images/lucas-card.jpg" alt="a black dog" onclick="alertDogInfo('Lucas', 'German Shepard', 123.45)">
```

Comment:
One way to improve this, is to create a function in your JS file. Then call this function from your HTML, passing in the data, and having the function handle the alert. It's a little overkill for this assignment, but good practice for passing arguments/params around.